data/types/settings/pump/blood_glucose_target_pre_meal_target.go

```diff
@@ -0,0 +1,42 @@
+package pump
+
+import (
+	"github.com/tidepool-org/platform/data"
+	dataBloodGlucose "github.com/tidepool-org/platform/data/blood/glucose"
+	"github.com/tidepool-org/platform/structure"
+)
+
+type BloodGlucosePreMealTarget struct {
```

Comment:
I think that this whole object could be replaced by the existing `dataBloodGlucose.Target` struct.

Reply:
Agree - never saw that one - but it works well - and less code - which is always better